doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
yo @Trott just add this bookmark, https://www.gnu.org/software/libc/manual/html_mono/libc.html 

`SIGPIPE` is a signal from the kernel. `EPIPE` is an error code. In the linked issue discussion I was talking more about termination signals (not `EPIPE` nor its conjoined twin, `SIGPIPE`)